tests/test_client.py

```diff
@@ -57,8 +57,8 @@ def set_up() -> Generator[Any, Any, Any]:
     )
     order_type = OrderType.LIMIT
     side = MarketSide.BUY
-    price = Price(amount=Decimal(50), currency=Currency.EUR)
-    quantity = Energy(mwh=Decimal(0.1))
+    price = Price(amount=Decimal("50"), currency=Currency.EUR)
+    quantity = Energy(mwh=Decimal("0.1"))
```

Comment:
Is this related to this PR?

Reply:
Yes because not having them in quotes resulted in the function to fail (converting floats to decimals like this seems to cause having e.g. 0.100000000000000000000000057901783 instead of 0.1).

Reply:
Ah, the niceties of floating point representation :D